cli/lib/kontena/cli/containers/exec_command.rb

```diff
@@ -10,9 +14,12 @@ def execute
       require_api_url
       token = require_token
 
-      payload = {cmd: ['sh', '-c', cmd_list.join(' ')]}
-      service_name = container_id.match(/(.+)-(\d+)/)[1]
+      cmd = build_command(cmd_list)
+      payload = {cmd: ["sh", "-c", cmd]}
+
+      service_name = container_id.match(/(.+)-(\d+)/)[1] rescue nil
```

Comment:
Not sure if we want to explicitly handle the missing match error here? This at least displays `Not found` instead of crashing hard.